build-system/tasks/runtime-test/index.js

```diff
@@ -617,9 +617,8 @@ async function runTests() {
         const result = browser.lastResult;
         // Prevent cases where Karma detects zero tests and still passes. #16851.
         if (result.total == 0) {
-          log(
-            red('ERROR: Zero tests detected by Karma. Something went wrong.')
-          );
+          log(red('ERROR: Zero tests detected by Karma.'));
+          log(red(JSON.stringify(result)));
```

Comment:
Should probably add a travis collapse here...

Reply:
It's just one line. Something like:
```
{"startTime":1558546493394,"total":4,"success":1,"failed":0,"skipped":3,"totalTime":23,"netTime":1,"error":false,"disconnected":false}
```

Reply:
It's just one line _that we know of now_ 😉

Reply:
But ok

Reply:
It's always going to be the one line.

https://github.com/karma-runner/karma/blob/master/lib/browser_result.js